### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by bratseth**
First half of fixing documentation

### CODE REVIEW

File: client/go/cmd/config.go [hunk 1]
```diff
@@ -55,7 +55,7 @@ var setConfigCmd = &cobra.Command{
 }
 
 var getConfigCmd = &cobra.Command{
-	Use:     "get option",
+	Use:     "get <option>",
```
Comment (mpolden) [inline client/go/cmd/config.go:58]: Should be `<[option]>` as omitting it prints all options.

File: client/go/cmd/deploy.go [hunk 2]
```diff
@@ -24,13 +24,13 @@ var (
 
 func init() {
 	rootCmd.AddCommand(deployCmd)
-	rootCmd.AddCommand(prepareCmd)
-	rootCmd.AddCommand(activateCmd)
+	// rootCmd.AddCommand(prepareCmd)
+	// rootCmd.AddCommand(activateCmd)
```
Comment (mpolden) [inline client/go/cmd/deploy.go:28]: Implemented in #18937.

Comment (bratseth) [reply client/go/cmd/deploy.go]: Right, fixed

File: client/go/cmd/config.go (hunk 1 above)
Comment (bratseth) [reply client/go/cmd/config.go]: done

**Comment by mpolden** [comment]
👍

**Pull request merged**